### ISSUE TITLE
Labels missing from Web (Crawler) Configuration 12.4

### ISSUE THREAD

**Issue by SoloRobo** [codelibs/fess#1914]
I've upgrade from 12.3.2 to 12.4, including elasticsearch and all the plugins.

Labels appears to have disappeared from the "Web Configuration" Crawler screen.

https://fess.codelibs.org/12.4/admin/webconfig-guide.html

Should I be using the Labels configuration differently?

https://fess.codelibs.org/12.4/admin/labeltype-guide.html

i.e. use the included and excluded paths in the Labels configuration screen instead of selecting a label in the Web Configuration screen?

I will attempt a fresh install to be sure it's not an issue related to upgrading.

**Comment by marevol** [comment]
> Should I be using the Labels configuration differently?

Yes. Upgrade process will remove labels from web configs in the next release.

**Comment by SoloRobo** [comment]
Thank you, I've superficially replicated the same config for Included and Excluded Paths in the Labels configuration from the Web (Crawler) Configuration.

Will there be a way to remove legacy Labels, or will future updates simply scrub that configuration?
